docs/inventory-plugins-templates.rst

```diff
@@ -0,0 +1,413 @@
+.. _ir_inv_plugin_templates_reference:
+
+Supported Inventory Plugin Templates
+==============================================
+
+.. index::
+    pair: templates;inventory plugins
+
+Upon upgrades, existing configurations will be migrated to the new format that will produce a backwards compatible inventory output. Use the templates below to help aid in migrating your inventories to the new style inventory plugin output.
+
+- :ref:`ir_plugin_aws_ec2`
+- :ref:`ir_plugin_gce`
+- :ref:`ir_plugin_azure`
+- :ref:`ir_plugin_vmware`
+- :ref:`ir_plugin_satellite`
+- :ref:`ir_plugin_openstack`
+- :ref:`ir_plugin_rhv`
+- :ref:`ir_plugin_rhaap`
+
+.. _ir_plugin_aws_ec2:
+
+Amazon Web Services EC2
+------------------------
+
+.. index::
+   pair: inventories; Amazon Web Services
+   pair: inventories; aws
+   pair: inventory plugins; aws
+
+::
```

Comment:
Since this PR is about moving code from one place to another, let's keep this. But a nice follow-up could be adding code highlighting, maybe.